examples/platform/efr32/wf200/host_if.cpp

```diff
@@ -15,6 +15,31 @@
  *    limitations under the License.
  */
 
+/***************************************************************************
+ * @file
```

Comment:
You are contributing this to the matter SDK after signing the CL and the matter licensing to Project CHIP Authors is already above.

Why is this additional license required?

Reply:
Hi Andy, thank you for the review and sorry for the mistake. Removed the extra license and made the commit.